[__main__.py] <2022-07-11 15:06:32.606094> pickle file-reading error ./dfdata/20220710/df_kicc_history_20220710 : [Errno 2] No such file or directory: './dfdata/20220710/df_kicc_history_20220710'
[__main__.py] <2022-07-12 08:25:21.665058> pickle file-reading error ./dfdata/20220711/df_kicc_history_20220710 : [Errno 2] No such file or directory: './dfdata/20220711/df_kicc_history_20220710'
[__main__.py] <2022-07-12 08:27:40.417765> pickle file-reading error ./dfdata/20220711/df_kicc_history_20220710 : [Errno 2] No such file or directory: './dfdata/20220711/df_kicc_history_20220710'
[__main__.py] <2022-07-12 08:27:44.069777> pickle file-reading error ./dfdata/20220711/df_kicc_history_20220710 : [Errno 2] No such file or directory: './dfdata/20220711/df_kicc_history_20220710'
[__main__.py] <2022-07-12 12:59:01.109309> pickle file-reading error ./dfdata/20220711/df_opera_trial_20220711 : [Errno 2] No such file or directory: './dfdata/20220711/df_opera_trial_20220711'
[__main__.py] <2022-07-12 12:59:56.130878> pickle file-reading error ./dfdata/20220711/df_opera_trial_20220710 : [Errno 2] No such file or directory: './dfdata/20220711/df_opera_trial_20220710'
[__main__.py] <2022-07-12 13:00:12.492924> pickle file-reading error ./dfdata/20220711/df_opera_trial_20220710 : [Errno 2] No such file or directory: './dfdata/20220711/df_opera_trial_20220710'
[edi_opera.py - Reading Data] <2022-07-14 14:39:09.423188> openpyxl file-reading error (./card_test.xlsx) ===> 'list' object is not callable
[edi_opera.py - Reading Data] <2022-07-14 14:44:02.490885> openpyxl file-reading error (./EDI-7월.xlsx) ===> list index out of range
[edi_opera.py - Reading Data] <2022-07-14 14:44:23.047046> openpyxl file-reading error (./EDI-7월.xlsx) ===> list index out of range
[edi_opera.py - Reading Data] <2022-07-14 14:45:25.101929> openpyxl file-reading error (./EDI-7월.xlsx) ===> list index out of range
[font_red_on_excel.py - Reading Data] <2022-07-15 08:41:47.161899> openpyxl file-reading error (./신용거래내역조회_220713.xlsx) ===> [Errno 2] No such file or directory: './신용거래내역조회_220713.xlsx'
[font_red_on_excel.py - Reading Data] <2022-07-15 08:42:22.050653> openpyxl file-reading error (./신용거래내역조회_220713.xlsx) ===> [Errno 2] No such file or directory: './신용거래내역조회_220713.xlsx'
[kicc_history.py - Making Excel] <2022-07-15 10:13:36.664225> Pandas.ExcelWriter error (df_kicc_receipts_.xlsx) ===> At least one sheet must be visible
[font_red_on_excel.py - Reading Data] <2022-07-15 11:01:07.469556> openpyxl file-reading error (C:/FA/creditcard/EDI_Confirm/신용거래내역조회_220713.xlsx) ===> [Errno 2] No such file or directory: 'C:/FA/creditcard/EDI_Confirm/신용거래내역조회_220713.xlsx'
[kicc_history.py - Making Excel] <2022-07-17 09:12:49.415318> Pandas.ExcelWriter error (./df_kicc_history_20220710) ===> [Errno 2] No such file or directory: './df_kicc_history_20220710'
[kicc_history.py - Making Excel] <2022-07-17 09:14:55.214960> Pandas.ExcelWriter error (./df_kicc_history_20220710) ===> Invalid extension for engine 'openpyxl': 'xlxs'
[kicc_history.py - Making Excel] <2022-07-17 09:14:56.983483> Pandas.ExcelWriter error (./df_kicc_history_20220710) ===> Invalid extension for engine 'openpyxl': 'xlxs'
[kicc_history.py - Making Excel] <2022-07-17 09:15:23.311404> Pandas.ExcelWriter error (./df_kicc_history_20220710) ===> Invalid extension for engine 'openpyxl': 'xlxs'
[kicc_history.py - Making Excel] <2022-07-17 09:16:21.757600> Pandas.ExcelWriter error (./df_kicc_history_20220710) ===> No engine for filetype: 'xlxs'
[kicc_history.py - Making Excel] <2022-07-17 09:16:43.174102> Pandas.ExcelWriter error (./df_kicc_history_20220710) ===> No engine for filetype: 'xlxs'
[kicc_history.py - Making Excel] <2022-07-17 09:17:08.433967> Pandas.ExcelWriter error (./df_kicc_history_20220710) ===> Invalid extension for engine 'openpyxl': 'xlxs'
[kicc_history.py - Making Excel] <2022-07-17 09:20:42.390878> Pandas.ExcelWriter error (./df_kicc_history_20220710) ===> Invalid extension for engine 'openpyxl': 'xlxs'
[kicc_history.py - Making Excel] <2022-07-17 09:21:20.134087> Pandas.ExcelWriter error (./df_kicc_history_20220710) ===> Invalid extension for engine 'openpyxl': 'xlxs'
[kicc_history.py - Making Excel] <2022-07-17 09:25:52.296692> Pandas.ExcelWriter error (./df_kicc_history_20220710) ===> Invalid extension for engine 'openpyxl': 'xlxs'
[kicc_history.py - Making Excel] <2022-07-17 09:30:17.335220> Pandas.ExcelWriter error (./df_kicc_history_20220710) ===> Invalid extension for engine 'openpyxl': 'xlxs'
[kicc_history.py - Making Excel] <2022-07-17 09:32:35.704840> Pandas.ExcelWriter error (./df_kicc_history_20220710) ===> Invalid extension for engine 'openpyxl': 'xlxs'
[kicc_history.py - Making Excel] <2022-07-17 09:33:00.120231> Pandas.ExcelWriter error (./df_kicc_history_20220710) ===> Invalid extension for engine 'openpyxl': 'xlxs'
[kicc_history.py - Making Excel] <2022-07-17 09:33:31.384717> Pandas.ExcelWriter error (./df_kicc_history_20220710) ===> Invalid extension for engine 'openpyxl': 'xlxs'
[kicc_history.py - Making Excel] <2022-07-17 09:33:53.164523> Pandas.ExcelWriter error (./df_kicc_history_20220710) ===> Invalid extension for engine 'openpyxl': 'xlxs'
[kicc_history.py - Making Excel] <2022-07-17 09:35:25.011310> Pandas.ExcelWriter error (./df_kicc_history_20220710) ===> No engine for filetype: 'xlxs'
[kicc_history.py - Making Excel] <2022-07-17 09:37:12.682140> Pandas.ExcelWriter error (./df_kicc_history_20220710) ===> [Errno 13] Permission denied: './dfdata/20220710/df_kicc_history_20220710.xlsx'
[kicc_history.py - Making Excel] <2022-07-17 09:38:47.144141> Pandas.ExcelWriter error (./df_kicc_history_20220710) ===> [Errno 13] Permission denied: './dfdata/20220710/df_kicc_history_20220710.xlsx'
[kicc_history.py - Making Excel] <2022-07-17 09:38:59.179587> Pandas.ExcelWriter error (./df_kicc_history_20220710) ===> [Errno 13] Permission denied: './dfdata/20220710/df_kicc_history_20220710.xlsx'
[kicc_history.py - Making Excel] <2022-07-17 09:39:07.080958> Pandas.ExcelWriter error (./df_kicc_history_20220710) ===> [Errno 13] Permission denied: './dfdata/20220710/df_kicc_history_20220710.xlsx'
[kicc_history.py - Making Excel] <2022-07-17 09:40:01.365047> Pandas.ExcelWriter error (./df_kicc_history_20220710) ===> [Errno 13] Permission denied: './dfdata/20220710/df_kicc_history_20220710.xlsx'
[kicc_history.py - Making Excel] <2022-07-17 09:42:34.548222> Pandas.ExcelWriter error (./df_kicc_history_20220710) ===> [Errno 13] Permission denied: './dfdata/20220710/df_kicc_history_20220710.xlsx'
[kicc_history.py - Making Excel] <2022-07-17 09:43:44.254172> Pandas.ExcelWriter error (./df_kicc_history_20220710) ===> [Errno 13] Permission denied: './dfdata/20220710/df_kicc_history_20220710.xlsx'
[kicc_history.py - Making Excel] <2022-07-17 09:44:35.427661> Pandas.ExcelWriter error (./df_kicc_history_20220710) ===> [Errno 13] Permission denied: './dfdata/20220710/df_kicc_history_20220710.xlsx'
[kicc_history.py - Making Excel] <2022-07-17 09:45:06.270453> Pandas.ExcelWriter error (./df_kicc_history_20220710) ===> [Errno 13] Permission denied: './dfdata/20220710/df_kicc_history_20220710.xlsx'
[kicc_history.py - Making Excel] <2022-07-17 09:46:15.195865> Pandas.ExcelWriter error (./df_kicc_history_20220710) ===> [Errno 13] Permission denied: './dfdata/20220710/df_kicc_history_20220710.xlsx'
[kicc_history.py - Making Excel] <2022-07-17 09:46:23.001109> Pandas.ExcelWriter error (./df_kicc_history_20220710) ===> [Errno 13] Permission denied: './dfdata/20220710/df_kicc_history_20220710.xlsx'
